testsuites/listener/shared/client_sg/test_changes.py

```diff
@@ -20,6 +20,12 @@ def test_longpoll_changes_termination_timeout(setup_client_syncgateway_test):
     4. Create another request GET /db/ on listener and make sure the listener responds
     """
 
+    liteserv_type = str(type(setup_client_syncgateway_test["liteserv"])).split(".")[1]
+    liteserv_version = str(setup_client_syncgateway_test["liteserv"].version_build)
```

Comment:
It would be nice if we had something similar to 
```
liteserv_platform = setup_client_syncgateway_test["liteserv"].platform
liteserv_version = setup_client_syncgateway_test["liteserv"]. version_build
```